### PR TITLE
added documentation for the eslint plugin

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Added docs for the eslint plugin

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 37141143ef4ed88190e518e601d2c2a12882c219 at: https://6114feaddb32edbe88848e46--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui.md [hunk 1]
```diff
@@ -73,6 +75,19 @@ The designs in this specification should result in enabling:
 
 - Meshery UI should be event-driven where possible.
 
+### Configurations
```
Comment (sudo-NithishKarthik) [inline docs/pages/project/contributing/contributing-ui.md:78]: Can we change this to `Linting-UI` and move it under `Setting up` heading ?

**Review by sudo-NithishKarthik — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 4a14c66af463c5e39e0ad3a33472d8de29385a07 at: https://611513bb2fe2a3b4b54ce613--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui.md [hunk 2]
```diff
@@ -86,10 +90,6 @@ The designs in this specification should result in enabling:
 - For Visual Studio Code, when using the `eslint` plugin, it automatically picks up the rules.
```
Comment (sudo-NithishKarthik) [inline docs/pages/project/contributing/contributing-ui.md:90]: I don't think this line is necessary too @warunicorn19

File: docs/pages/project/contributing/contributing-ui.md (hunk 1 above)
Comment (sudo-NithishKarthik) [reply docs/pages/project/contributing/contributing-ui.md]: Thank you @warunicorn19  😄

**Comment by l5io** [comment]
🚀 Preview for commit d0f11e94ba26947394c7a514bdb5edb961a02fae at: https://611516e2d0cfb4b7ddff8640--meshery-docs-preview.netlify.app

**Pull request merged**